src/components/fullscreen-spinner.tsx

```diff
@@ -0,0 +1,9 @@
+export default function FullscreeenSpinner() {
```

Comment:
when i try to render the component from the homepage, it shows up on the left side of the screen for me. could you make it so that it is always above everything, centered, and also blurs out the rest of the page?

also small spelling mistake in the component name